stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/EventHandlerImpl.kt

```diff
@@ -209,19 +209,23 @@ internal class EventHandlerImpl(
                 // note that many of these events should also update user information
                 is NewMessageEvent -> {
                     event.message.enrichWithCid(event.cid)
+                    event.message.enrichWithOwnReactions(batch, event.user)
                     event.totalUnreadCount?.let { domainImpl.setTotalUnreadCount(it) }
                     batch.addMessageData(event.cid, event.message)
                 }
                 is MessageDeletedEvent -> {
                     event.message.enrichWithCid(event.cid)
+                    event.message.enrichWithOwnReactions(batch, event.user)
                     batch.addMessageData(event.cid, event.message)
                 }
                 is MessageUpdatedEvent -> {
                     event.message.enrichWithCid(event.cid)
+                    event.message.enrichWithOwnReactions(batch, event.user)
```

Comment:
When message is updated by other user, the event contains `ownReactions` of other user. Therefore, for such events we use previous `ownReactions` stored in the database